ariadne_codegen/settings.py

```diff
@@ -53,6 +53,7 @@ class ClientSettings(BaseSettings):
     include_comments: bool = True
     convert_to_snake_case: bool = True
     async_client: bool = True
+    unwrap_single_fields: bool = False
```

Comment:
As said [here](https://github.com/mirumee/ariadne-codegen/issues/66#issuecomment-1545578003) let's remove this setting and add information about plugin to `README.md`

Reply:
Removed settings and added section to readme. Not sure how you want to structure this, I just added it in the root with a short summary of what it does, linking to it for further docs.